akka-stream-tests/src/test/scala/akka/stream/impl/fusing/InterpreterSpec.scala

```diff
@@ -550,6 +550,8 @@ class InterpreterSpec extends AkkaSpec with GraphInterpreterSpecKit {
 
     }
 
+    //#20386
+    @deprecated("To be removed from 2.5, when all stages become GraphStage", "2.4.5")
```

Comment:
Maybe a different message: "Usage of PushPullStage is deprecated, please use GraphStage instead".